azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineImages.java

```diff
@@ -28,6 +28,18 @@
     VirtualMachineImage getImage(Region region, String publisherName, String offerName, String skuName, String version);
 
     /**
+     * Gets a virtual machine image.
+     *
+     * @param region the region
+     * @param publisherName publisher name
+     * @param offerName offer name
+     * @param skuName SKU name
+     * @param version version name
+     * @return the virtual machine image
+     */
+    VirtualMachineImage getImage(String region, String publisherName, String offerName, String skuName, String version);
```

Comment:
It's not the first time actually, there used to be a similar method in the early Betas, VirtualMachineImagesOperations.get().